modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/ImportTranslationMVCActionCommand.java

```diff
@@ -262,6 +337,9 @@ private void _importZipFile(
 		}
 	}
 
+	private Map<String, String> _importErrors;
```

Comment:
These cannot be instance variables. The same action command will be shared by multiple (potentially simultaneous) requests, so this could wreck havoc. Please keep everything as local as possible; if you need to communicate across multiple requests, use the session.

Reply:
Thanks!
I did not realize it when extracted the methods.

Thanks for pointing at it.


rebasing and sending